src/python/gem5/prebuilt/viper/viper_network.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) 2021 The Regents of the University of California.
```

Comment:
Probably you want this to be AMD instead of Davis, @abmerop ?

Reply:
Jason wrote the Pt2Pt class in this file so I kept the copyright